lib/storage/sfSessionStorage.class.php

```diff
@@ -190,5 +190,6 @@ public function shutdown()
   {
     // don't need a shutdown procedure because read/write do it in real-time
     session_write_close();
+    self::$sessionStarted = false;
```

Comment:
Is this needed for PHP 8.0 & 8.1 or is just a good idea to do?

Reply:
Please see the next comment ^^